woocommerce/woocommerce-android
# Issue/6069 order creation product auto search

Closes #6069 - This PR changes the product search in order creation to search as you type, so tapping the search button is not required (but is still supported).

To test:

* Start order creation
* Tap to add a product
* Search the product list and ensure it search as you type (with a brief delay) and everything works as expected.

https://user-images.githubusercontent.com/3903757/159551347-b9c4f54e-9221-4c55-baf0-e15e0eebb067.mp4
